pyaedt/maxwell.py

```diff
@@ -2751,7 +2814,7 @@ def assign_vector_potential(self, input_edge, vectorvalue=0, bound_name=None):
     def assign_master_slave(
         self, master_edge, slave_edge, reverse_master=False, reverse_slave=False, same_as_master=True, bound_name=None
     ):
-        """Assign master and slave boundary conditions to two edges of the same object.
+        """Assign Dependent and Independent boundary conditions to two edges of the same object.
```

Comment:
```suggestion
        """Assign dependent and independent boundary conditions to two edges of the same object.
```
Put independent first?